lib/Slider/Slider.js

```diff
@@ -898,6 +898,7 @@ module.exports = kind(
 	*/
 	previous: function () {
 		this.set('value', this.value - this._jumpIncrementAmount);
+		if (this.popup && this.$.popup.getShowing()) return true;
```

Comment:
Hi @DavidUM, thanks for your update. I wasn't clear - is there a specific reason we want to allow the `onSpotlightSelect` event to bubble up when pressing one of the jump increment buttons, as opposed to always returning true here?